components/Portal/JournalEntry/index.tsx

```diff
@@ -1,25 +1,29 @@
 import styles from "./journalentry.module.scss";
-import {EntryProp} from "contentful-management/dist/typings/entities/entry";
```

Comment:
Didn't realize there was a frontend component for this already - changing the return type probably messed this function up. Thanks for fixing it though!